.homebridge-dev/config.json

```diff
@@ -31,7 +31,8 @@
       "deviceManagement": {
         "blacklistOrWhitelist": "blacklist",
         "blacklistedUniqueIDs": [""]
-      }
+      },
+      "uniqueNames": true
```

Comment:
Change the name to something representing mac address. i.e. MACaddressNames, additionally place it under a category or create a new category.

Reply:
done. Changed to namesWithMacAddress, and added into a category.